addons/project/models/project_project.py

```diff
@@ -13,6 +13,9 @@
 from .project_update import STATUS_COLOR
 from .project_task import CLOSED_STATES
 
+AccessWarningMessage = _(
+                    "The project's limited privacy prevents sharing. Please make it public so collaborators can access it.")
```

Comment:
Don't do that, the translation will be only done based on the language of OdooBot instead of the current since the function will be called when we start the server. 😬 

cc @auon-odoo